### PR TITLE
allow deleting of options

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit review please

File: unicore/ask/service/questions_api.py [hunk 1]
```diff
@@ -76,9 +76,17 @@ def put(self):
             if value is not None and not attr == 'options':
                 setattr(question, attr, value)
 
-        for option in self.request.validated.get('options', []):
+        validated_options = self.request.validated.get('options', [])
+        # Delete existing options
+        if question.question_type != 'free_text':
+            existing_options = [
```
Comment (Rizziepit) [inline unicore/ask/service/questions_api.py]: What happens if a question is changed from `multiple_choice` to `free_text`?

Related question: do we want to delete options altogether, or maybe just delete the relation? I'm thinking of the case where there is existing answers for old options.

**Comment by miltontony** [comment]
@Rizziepit I removed the check for `free_text` so it will work like the rest.

**Comment by Rizziepit** [comment]
+1

**Pull request merged**